### PR TITLE
Updated README.md - Files required to connect to DBs and instructions…

### PR DESCRIPTION
**Pull request by johnnyfwk**
… to create environment variables.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,9 @@
...
-git branch -D <local branch>
-```
-
-## Husky
-
-To ensure we are not commiting broken code this project makes use of git hooks. Git hooks are scripts triggered during certain events in the git lifecycle. Husky is a popular package which allows us to set up and maintain these scripts. This project makes use a _pre-commit hook_. When we attempt to commit our work, the script defined in the `pre-commit` file will run. If any of our tests fail than the commit will be aborted.
-
-The [Husky documentation](https://typicode.github.io/husky/#/) explains how to configure Husky for your own project as well as creating your own custom hooks.\_
+The files a developer must add in order to successfully connect to the two databses locally are:
+- ./db/connection.js
```
Comment (popatre) [inline README.md:2]: You will add these files, so someones using the repo doesn't need to know or do this

File: README.md [hunk 2]
```diff
@@ -1,57 +1,9 @@
...
+- ./db/data/development-data/index.js
+- ./db/data/test-data/index.js
+
+To create the environment variables:
+- set ENV to process.env.NODE_ENV || 'development' then append ENV
```
Comment (popatre) [inline README.md:7]: This bit, will all be done by you, they wont need to do it. 

you need to explain how to make the .env files i.e. `create a file called .env.test`

Then what they need to put inside those files. 
be specific to make it as easy as possible.

**Review by popatre — CHANGES_REQUESTED** [review]
Please see my comments

**Review by Poonam-raj — APPROVED** [review]
Env instructions are very clear, nice one

**Pull request merged**